### ISSUE TITLE
Update Docker image 

### ISSUE THREAD

**Issue by clintonb** [GoogleCloudPlatform/gke-autoneg-controller#122]
https://github.com/googlecloudplatform/gke-autoneg-controller/pkgs/container/gke-autoneg-controller%2Fgke-autoneg-controller/136007887?tag=v1.0.0 is very much outdated compared to the repo. I wasted a lot of time trying to figure out why I couldn't use the `--auto-reconcile` flag before realizing the image predates the README.

Please release a new version of the image, and update the various code locations to use this newer version. Otherwise, update the README to better reflect the availability of various parameters in the "current" image.

**Comment by rosmo** [comment]
I'm not sure about this, it's kind of the point that it's pinned to the last "major release". You are of course free to use the bleeding edge, but I'm not sure if I want to impose that on everyone else.

**Comment by clintonb** [comment]
@rosmo how do release work for this project? It's been six months since v1.0.0, but there have been a number of Helm chart releases. Why not produce a _minor_ release (e.g., v1.1.0) with the new functionality? I don't want to use the "bleeding edge," either.

My major issue is the huge gap between the README and the currently-published image. There needs to be a better callout that users will need to either build their own images or be comfortable on the bleeding edge with `latest`. Pinning to a six month old image is frustrating because it wastes precious time for folks (me) who expect the README instructions to either "just work" or highlight potential pitfalls.

**Comment by clintonb** [comment]
@rosmo here is an alternative: https://github.com/GoogleCloudPlatform/gke-autoneg-controller/pull/124

**Comment by rosmo** [comment]
I appreciate the feedback. This is a open source, best effort basis product (having said that, it works well and based on statistics, most people aren't even on v1.0.0). 

I anticipate we will release a new version once the `capacityScaler` issues has been addressed (might be one of two other small things). It will be a "breaking change" (as some of the details of the implementation will change) and probably versioned as `v1.1.0`).

But, I have merged #124 now as it's a good callout.
